frontend/src/Components/Countdown/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { DAY_MILLIS, HOUR_MILLIS, MINUTE_MILLIS, SECOND_MILLIS } from '~/constants';
+
+export const calculateTimeLeft = (targetDate: Date): string => {
```

Comment:
Helst unngå arrow functions. Det er faktisk mer verbost og mindre lesbart enn enkel `function`
```suggestion
export function calculateTimeLeft(targetDate: Date): string {
```

Reply:
Sure, done